torch/_refs/__init__.py

```diff
@@ -355,12 +358,31 @@ def _ref(a: TensorLikeType) -> TensorLikeType:
     type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT,
 )
 
+exp2 = _make_elementwise_unary_reference(
+    prims.exp2,
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT,
+    aten_op=torch.ops.aten.exp2,
+)
+
 floor = _make_elementwise_unary_reference(
     prims.floor,
     type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT,
 )
 
 
+def _frac(x: TensorLikeType) -> TensorLikeType:
+    # frac(x) = x - floor(abs(x)) * sign(x)
+    trunc_x = mul(floor(abs(x)), sign(x))
+    return sub(x, trunc_x)
+
+
+frac = _make_elementwise_unary_reference(
+    _frac,
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT,
+    aten_op=torch.ops.aten.frac,
```

Comment:
Same here -- I think we can get away without this?

Reply:
I think this is required, since `aten_op` won't be inferred with `_frac`.

Reply:
Oh you're totally right -- sorry, I didn't notice the underscore